kovri.md

```diff
@@ -2,57 +2,63 @@ Kovri Lightweight C++ I2P Router Script
 
-In a sentence, Kovri is a lightweight router written in C++, compatible with cryptocurrencies, and built for the I2P network. 
+In a sentence, Kovri is a lightweight router compatible with cryptocurrencies and built for the I2P network.
 I2P stands for Invisible Internet Protocol.
-I2P is a layer of the internet that allows for all connections in the network to be private. 
+I2P is a layer of the internet that allows for all connections in the network to be private.
 In contrast with regular internet routing or TOR's onion routing, I2P utilizes what it refers to as garlic routing.
-Instead of connecting with participants directly or through TOR nodes, such as going directly to a website, all information is encrypted and relayed through other I2P routers in the network. 
+Instead of connecting with participants directly or through TOR nodes, such as going directly to a website, all information is encrypted and relayed through other I2P routers in the network.
 
 Since I2P routers know little or no information about the data they are sending, where it originated, or its destination, all information in this network is private and anonymous.
-Nodes that log Monero traffic will now get a meaningless .i2p address, which has no connection to a node's real IP address.
-Thus, even if nodes continue to log information, they will not learn anything from passive surveillance.
-Now, when Alice sends her transaction through I2P with the Kovri router, an attacker is no longer able to associate her IP address with her transaction. 
+If Alice sends her transaction with Kovri, nodes that log Monero traffic will no longer get her real IP address, rendering these passive surveillance tactics useless.
+
+Now let’s imagine a different scenario.
+Suppose Alice has an internet service provider that does not want her to use Monero, who send her a warning saying so.
+Alice can optionally send all the Monero sync data, not just the transaction broadcast, through I2P.
+Now, her Monero connections are hidden from her ISP.
+This improves Monero's censorship resistance, since other people no longer know who is running a Monero node and demand that they be shut down.
 
-Furthermore, suppose Alice has an internet service provider that does not want her to use Monero. 
-Alice can optionally send all the Monero sync data, not just the transaction broadcast, through I2P. 
-Now, her ISP will not know that she is using Monero, improving Monero's censorship resistance.
+Let’s look at one last scenario.
+Suppose George is the operator of a mining pool that controversially donates a portion of the block reward to a political party.
+Other nodes could purposefully reject his solved blocks to express their disagreement with this policy.
+This sort of attack is called a Sybil attack.
+However, if George can send his solved blocks through Kovri, then no one can censor the work from his pool, since they no longer know that his pool created it.
```

Comment:
>George

Alice, Bob, Charlie, Dave, Eve...

These names have a purpose, and I'm assuming that you picked a name at random? In this scenario, Charlie is more accurate (though Dave or Frank could be argued as well).

https://en.wikipedia.org/wiki/Alice_and_Bob